app/decorators/podcast_episode_decorator.rb

```diff
@@ -25,4 +25,12 @@ def published_timestamp
 
     published_at.utc.iso8601
   end
+
+  def mobile_player_metadata
+    {
+      podcastName: podcast.title,
+      episodeName: title,
+      podcastImageUrl: ApplicationController.helpers.app_url(podcast.image_url)
```

Comment:
Now that I think about it, shouldn't this be simply `.image_url`? 
If you look at 

https://github.com/thepracticaldev/dev.to/blob/db4c9ef4b23fe2c531e7d00733b6d4ca08ece181/app/views/podcast_episodes/_liquid.html.erb#L20-L29

we render the `podcast.image_url` directly (the reason it's confusing is that locally the `image_url` is a path, only because we don't have the domain setup) but in production is the URL of the uploaded image.

In addition, as you see in the example, the image is resized and enhanced to fit specific parameters, which you might want to replicate for the mobile (the uploaded image could be 1000 pixels :D).

In Ruby code you can use the `cloudinary()` helper to achieve the same:

https://github.com/thepracticaldev/dev.to/blob/db4c9ef4b23fe2c531e7d00733b6d4ca08ece181/app/helpers/application_helper.rb#L63-L77

Reply:
Thanks for this @rhymes! You're absolutely right since in Prod we do get a full URL it would work. I wasn't aware of this and of course locally didn't work for me.

I just pushed an update that includes the cloudinary helper. I'm using a `600` width image because devices have some pretty big pixel density and wouldn't want to compromise the image quality ([based on this discussion](https://discussions.apple.com/thread/6521841)).

The image will be served on demand regardless of the size requested right? Meaning there's no need to configure the image to be scaled to the `600` size? I'm just not aware of the best usage or inner workings of Cloudinary and can't think of a way to test this until it hits production 😅

(researching about it too on my end too in the mean time)